Makefile

```diff
@@ -1,6 +1,7 @@
 .DEFAULT_GOAL := all
-isort = isort toucan_connectors tests setup.py
-black = black toucan_connectors tests setup.py
+isort = isort toucan_connectors tests
+black = black toucan_connectors tests
+flake8 = flake8 toucan_connectors tests
```

Comment:
We don't do it in laputa, and I find it more constraining: 
* If the user wants to manage their venvs themselves rather than delegating that to poetry (which most of us have to do to juggle between python versions), having `poetry run` is annoying.
* If the user does want to delegate venv creation to poetry, they can easily activate it before running `make` commands, either with manual activation or with `poetry shell`